### PR TITLE
OSDOCS#9760: Adds management console known issue to 4.15 RNs

### PR DESCRIPTION
**Pull request by opayne1**
Adds management console known issue https://issues.redhat.com/browse/OCPBUGS-29812 to 4.15 RNs

Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9760

Link to docs preview:
https://72060--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-known-issues

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2578,6 +2578,8 @@ If NMEA sentences are lost on their way to the e810 NIC, the T-GM cannot synchro
 A proposed fix is to report a `FREERUN` event when the NMEA string is lost.
 (link:https://issues.redhat.com/browse/OCPBUGS-19838[*OCPBUGS-19838*])
 
+* Currently, the YAML tab of some pages in the web console will stop unexpectedly on some browsers when the multicluster engine for Kubernetes operator (MCE) is installed. An error notification "`TypeErrorDescription:e` is undefined`" is displayed. https://issues.redhat.com/browse/OCPBUGS-29812[*OCPBUGS-29812*]
```
Comment (yapei) [inline release_notes/ocp-4-15-release-notes.adoc:2581]: The error message is a little different, can we update? attaching a screenshot for reference
Screenshot 2024-02-23 at 3 39 56 PM

Comment (opayne1) [reply release_notes/ocp-4-15-release-notes.adoc]: Thank you for the screenshot! I have updated the note.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -2591,8 +2591,14 @@ If NMEA sentences are lost on their way to the e810 NIC, the T-GM cannot synchro
 A proposed fix is to report a `FREERUN` event when the NMEA string is lost.
 (link:https://issues.redhat.com/browse/OCPBUGS-19838[*OCPBUGS-19838*])
 
+* Currently, the YAML tab of some pages in the web console will stop unexpectedly on some browsers when the multicluster engine for Kubernetes operator (MCE) is installed, and displays the message "Oh no! Something went wrong." https://issues.redhat.com/browse/OCPBUGS-29812[*OCPBUGS-29812*]
+
 //[id="ocp-telco-ran-4-15-known-issues"]
 //
+
+
+[id="ocp-telco-ran-4-15-known-issues"]
```
Comment (kalexand-rh) [inline release_notes/ocp-4-15-release-notes.adoc:2600]: This line should be removed - it's commented out on line 2596.

**Review by kalexand-rh — APPROVED** [review]
This is fine to merge.

**Review by skrthomas — COMMENTED** [review]
A couple small things. LGTM otherwise.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -2591,8 +2591,14 @@ If NMEA sentences are lost on their way to the e810 NIC, the T-GM cannot synchro
 A proposed fix is to report a `FREERUN` event when the NMEA string is lost.
 (link:https://issues.redhat.com/browse/OCPBUGS-19838[*OCPBUGS-19838*])
 
+* Currently, the YAML tab of some pages in the web console will stop unexpectedly on some browsers when the multicluster engine for Kubernetes operator (MCE) is installed, and displays the message "Oh no! Something went wrong." https://issues.redhat.com/browse/OCPBUGS-29812[*OCPBUGS-29812*]
```
[the remark this answers is not in the document]
Comment (opayne1) [reply release_notes/ocp-4-15-release-notes.adoc]: Thanks!

**Comment by JoeAldinger** [comment]
Merge requested via slack.

**Pull request merged**